src/main/java/edu/kis/vh/nursery/stacks/IntArrayStack.java

```diff
@@ -13,28 +11,28 @@ public int getTotal() {
 
     private int total = ERROR_CODE;
 
-    public void countIn(int in) {
+    public void push(int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
-    public boolean callCheck() {
+    public boolean isEmpty() {
         return total == ERROR_CODE;
     }
 
     public boolean isFull() {
         return total == 11;
     }
 
-    protected int peekaboo() {
-        if (callCheck())
-            return ERROR_CODE;
+    public int top() {
+        if (isEmpty())
+            return ERROR_CODE+1;
```

Comment:
It should be enough to change a value of the constant, otherwise, it does not make sense.